app/assets/stylesheets/components/_login.scss

```diff
@@ -1,49 +1,66 @@
 .__content {
   background-color: $background_color;
   margin-top: 12.5%;
-  text-align: center;
 
   .mid{
     display: grid;
     font-family: $poppins;
-    gap: 2rem;
+    gap: 3rem;
     grid-template-columns: 1fr 1fr;
     margin: 0 auto;
-    max-width: 50rem;
+    max-width: 80rem;
+    text-align: center;
 
     .login{
       background-color: $modal_color;
-      border-radius: 5%;
-      padding:1rem 0rem;
+      border-radius: 1.5rem;
+      display: grid;
+      gap: 2rem;
+      padding: 2rem;
+
+      div{
+        display: grid;
+        gap: 1rem;
+      }
 
       .input{
         background-color: $background_color;
-        border-radius: 5%;
-        font-size: 1.2rem;
+        border: none;
+        font-size: 2rem;
         height: 3.5rem;
-        margin-top: 2rem;
         padding-left: 1rem;
-        width: 17rem;
       }
 
       .button{
         background-color: $button_color;
+        border: none;
         color: $color_white;
+        font-size: 1.5rem;
         height: 3.5rem;
-        margin-top: 2rem;
-        width: 17rem;
       }
 
-      .button__sigIn{
-        background-color: $feed_card;
-        color: $color_white;
-        height: 3.5rem;
-        margin-top: 2rem;
-        width: 17rem;
+      .link-container{
+        margin-top: -0.5rem;
+        .button__sigIn{
```

Comment:
this is a bem nomenclature, please use it or change the name of this className

Reply:
done